proposals/0000-simd-process.md

```diff
@@ -0,0 +1,170 @@
+
+There are two types of proposals:
+
+- A **Standard** Proposal describes any change that affects most or all Solana
+implementations, such as a change to the network protocol, consensus, proposed
+application standards/conventions, block or transaction validity, or any change
+or addition that affects the interoperability of applications using Solana.
+Standard proposals can be broken down into the following categories:
+
+  - **Core**: Anything that affects consensus or substantial changes to the
+validator.
+  - **Networking**: Changes or substantial improvements to network protocol
+specifications.
+  - **Interfaces**: Improvements around the client JSON RPC API specifications
+and standards. 
+
+- A **Meta** Proposal describes a process surrounding Solana or proposes a
+change to (or an event in) a process. Process Proposals are like Standard
+Proposals but apply to areas other than the Solana protocol itself. They may
+propose an implementation, but not to Solana's codebase; they often require
+community consensus; unlike Informational Proposals, they are more than
+recommendations, and users are typically not free to ignore them. Examples
+include procedures, guidelines, changes to the decision-making process, and
+changes to the tools or environment used in Solana development. Any meta-EIP is
+also considered a Process Proposals.
+
+## Proposal Lifecycle
+
+The stages in a lifecycle of a proposal are as follows:
+
+- Idea
+- Draft
+- Review
+- Accepted
+- Stagnant
+- Withdrawn
+- Implemented
+
+```mermaid
+flowchart LR
+  Idea
+  Draft
+  Review
+  Accepted
+
+  subgraph fail[&nbsp];
+    Stagnant
+    Withdrawn
+  end
+
+  style fail fill:#ffe8e7,stroke:none
+  style Accepted fill:#f3f8dc,stroke:#c3db50;
+
+  Idea ---> Draft;
+  Draft ---> Review;
+  Review ---> Accepted;
+
+  Draft ---> Stagnant;
+  Review ---> Stagnant;
+  Review ---> Withdrawn;
+```
+
+### Idea
+
+At the idea stage, parties involved in the proposal are you, the champion or
+proposal author, the reviewers, and the Solana Core Contributors.
+
+Before you begin writing a formal proposal, you should vet your idea. Ask the
+Solana core community first if an idea is original to avoid wasting time on
+something that will be rejected based on prior research. It is thus recommended
+to discuss the proposal on the Solana Tech Discord under the #core-technology
+channel.
+
+### Draft
+
+To begin drafting the proposal, do the following:
+
+- Fork the proposal repository
+- Copy `proposals/XXXX-template.md` to `proposals/XXXX-my-feature.md` (where
+"my-feature" is descriptive)
+- Fill in the proposal. Put care into the details: proposals that do not
+present convincing motivation, demonstrate lack of understanding of the
+design's impact, or are disingenuous about the drawbacks or alternatives tend
+to be poorly received.
+- Submit a pull request.
+- Now that your proposal has an open pull request, use the issue number of the
+PR to update the `XXXX-` prefix to the number.
```

Comment:
Following these instructions, this should be 0001

Reply:
Updated based on instructions